pages.pl/linux/apt-add-repository.md

```diff
@@ -0,0 +1,20 @@
+# apt-add-repository
+
+> Zarządza definicjami repozytoriów apt.
```

Comment:
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases

Seems like the English page is written in the same manner - I'll update it later.
```suggestion
> Zarządzaj definicjami repozytoriów `apt`.
```